PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -29,12 +29,14 @@
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
+    <NoWarn>CS2008</NoWarn>
   </PropertyGroup>
   <PropertyGroup>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
btw @duncanpMS I was wondering the other day... how do we reference NewtonSoft? We don't seem to be using NuGet